### PR TITLE
create workbenches

### PR DESCRIPTION
**Pull request by ranjithamca**

### CODE REVIEW

**Comment by indykish** [comment]
@rajthilakmca  Please review and merge it.

File: app/models/json/analytics/WorkbenchesSerialization.scala [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+ * @author ranjitha
+ *
+ */
+class WorkbenchesSerialization(charset: Charset = UTF8Charset) extends models.json.SerializationBase[Workbenches] {
+
+  protected val NameKey = "name"
+  protected val ConnectorsKey = "connectors"
+
+
+  override implicit val writer = new JSONW[Workbenches] {
```
Comment (rajthilakmca) [inline app/models/json/analytics/WorkbenchesSerialization.scala:44]: please remove the unwanted spaces

Comment (rajthilakmca) [inline app/models/json/analytics/WorkbenchesSerialization.scala:44]: please remove the unwanted spaces

File: app/controllers/analytics/Workbenches.scala [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+/**
+ * @author ranjitha
+ *
+ */
+
+object Workbenches extends Controller with controllers.stack.APIAuthElement {
+
+  /**
+   * Create a new sparkjobs entry by email/json input.
+   */
```
Comment (rajthilakmca) [inline app/controllers/analytics/Workbenches.scala:37]: add get method for workbenches controller

**Pull request merged**